cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<p>Sometimes it is useful to modify the data stored in a cooler file. </p><p> </p><p>This can include:</p><ul><li>removing chromosomes</li><li>renaming chromosomes</li><li>re-ordering chromosomes</li></ul><p> </p><p>This notebook provides examples of how to achieve these operations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>